src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -71,6 +72,9 @@ public static class DicomServerServiceCollectionExtensions
                 options.EnableEndpointRouting = false;
                 options.RespectBrowserAcceptHeader = true;
                 options.OutputFormatters.Insert(0, new DicomJsonOutputFormatter());
+            }).AddJsonOptions(jsonOptions =>
+            {
+                jsonOptions.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
             });
```

Comment:
nit: Given that we're not chaining together calls elsewhere, let's just use:

```csharp
services.AddJsonOptions(jsonOptions => ... )
```

Reply:
Hill, it's chained... AddJsonOptions is a method of IMvcBuilder which returns from previous call.